docs/source/metoffice.rst

```diff
@@ -0,0 +1,165 @@
+.. _MetOffice:
+
+Using Fab at the Met Office
+===========================
+
+For users working on Met Office desktop systems the following instructions may
+prove useful.
+
+Using Fab
+~~~~~~~~~
+
+If you just want to use Fab to build some source then there are a number of
+fairly simple approaches.
+
+
+Fortran Only
+------------
+
+If you don't need to build C, you can use this the modules in the "extra
+software" collection:
+
+.. code-block:: console
+
+    $ module use $EXTRA_SOFTWARE
+    $ module load python/3.7.0 support/fparser/0.16.0/python/3.7.0 fab/0.11.0/python/3.7.0
```

Comment:
This `module load` command (`module load python/3.7.0 support/fparser/0.16.0/python/3.7.0 fab/0.11.0/python/3.7.0`) does not seem to work for me. I receive the following error(s): 

ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'python/3.7.0'
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'support/fparser/0.16.0/python/3.7.0'
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'fab/0.11.0/python/3.7.0'

May I be missing something here? I managed to successfully execute `module use $EXTRA_SOFTWARE` (with the correct path, provided by Giorgia).

Reply:
Not sure why Python didn't load, it works for me. fParser had the wrong version number,  not fixed.

I notice that Fab is still the beta version so I've updated it to 1.0.0 but I'm still waiting for that module to be reviewed so it doesn't exist yet. The beta module listed here does exist so I'm not sure why it fails.